parasolr/query/aliased_queryset.py

```diff
@@ -110,6 +116,16 @@ def get_facets(self) -> Dict[str, int]:
 
         return facets
 
+    def get_stats(self) -> Dict[str, Dict]:
+        """Extend :meth:`parasolr.query.queryset.SolrQuerySet.get_stats` to
+        return return aliased field names for field_list keys."""
+        stats = super().get_stats()
+        for field in stats['stats_fields']:
+            stats['stats_fields'][self.reverse_aliases.get(field, field)] \
+                = stats['stats_fields'].pop(field)
```

Comment:
I wonder if a dict comprehension would be more readable here? Implementation might also depend what we decide on modifying the dict by value.

Reply:
if nothing else, it'd be consistent with `get_facets`, so I'll go with the established precedent.